charts/opensearch/values.yaml

```diff
@@ -196,11 +196,6 @@ extraContainers: []
   #   image: busybox
   #   command: ['do', 'something']
 
-extraInitContainers: []
```

Comment:
We should keep this @alborotogarcia

Reply:
I forgot to add it back nice catch @DandyDeveloper !